node/runtime/src/lib.rs

```diff
@@ -60,8 +60,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node"),
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
-	spec_version: 30,
-	impl_version: 30,
+	spec_version: 31,
```

Comment:
You don't touch the runtime, so you don't need to change the `spec_version` and `impl_version`.